packages/eset_protect/manifest.yml

```
@@ -1,7 +1,7 @@
 format_version: 3.0.3
 name: eset_protect
 title: ESET PROTECT
-version: "1.6.1"
+version: "1.6.2"
```

```suggestion
version: "1.7.0"
```
This should match changelog version.